common/check_used_space

```diff
@@ -8,19 +8,33 @@
 # Authors:
```

Comment:
Here again I would make a new copy which is used for monitoring, check_report_used_space